docs/cheaha/open_ondemand/index.md

```diff
@@ -25,3 +25,40 @@ For a more complete description of how to select resources, go [here](ood_layout
+1. They both support `.ipynb` notebooks.
+1. Allow interactive code execution, plotting, markdown, and data visualization.
+1. Can run Python, R, Julia, and other supported languages.
+1. they are both great for reproducing workflows.
+1. You can access both on Cheaha, selecting the configuration and resources required for your workflow.
+
+### Why Choose JupyterLab?
+
+JupyterLab is the next-generation interface for Jupyter. It's designed to be a full-featured web-based IDE. JupyterLab provides an interface for interactive and exploratory computing. Although it includes many features typical of traditional integrated development environments (IDEs), its primary focus remains on interactive and exploratory computing. The following are reasons why you may use JupyterLab over Jupyter Notebook.
+
+1. JupyterLab will be your preferred app, if you are conversant with or your workflow requires you to manage multiple notebooks, terminals, and other files. JupyterLab allows you to in a single tabbed interface, run your code, analyze logs, edit documentation, and move between tasks within your workflow more quickly.
+1. Projects that involve code, markdown, output files, and terminals simultaneously can be split into tabs within a browser tab, without switching browser tabs.
+1. If you want a more customizable and extensible environment, JupyterLab supports extensions out-of-the-box. If you need a Git GUI or a table of contents for navigating long notebooks? JupyterLab lets you add these features easily, making it more like a lightweight IDE than just a notebook viewer.
+
+![!Screenshot of JupyterLab interface with notebook + terminal + markdown viewer open side-by-side.](./images/jupyterlabsample.png)
+
+### Why Choose Jupyter Notebook?
+
+Jupyter Notebook offers the classic, streamlined experience that focuses on a single document at a time, the Jupyter Notebook interface is  ideal for teaching, learning, or lightweight research tasks. This interface, is what many users are already familiar with. The following are reasons why you may use Jupyter Notebook over JupyterLab.
+
+1. If you are a beginner or a user focusing on a single notebook or analysis for your computational workflow, the simplicity of Jupyter Notebook will serve you best. As there is very little to learn compared to JupyterLab, there are fewer buttons to click to complete your tasks.
+1. If you are teaching a class, or developing tutorials, or workshops where simplicity is a priority. Jupyter Notebooks will help you minimize distractions, with your audience staying focused, writing and running their code, and not navigating a complex interface like with JupyterLab.
+1. Jupyter Notebooks are also excellent for running very quick exploratory data analysis, you can test out sample code and data, develop quick plots. Without all of the additional complexities JupyterLab may need you to do while setting it up.
```

Comment:
Split first sentence at first comma. Avoid second person pronoun "you". Describe what is possible, rather than what "you" can do.

Reply:
Last sentence is a fragment.